catalog/core/catalog-core-urlresourcereader/src/test/java/ddf/catalog/resource/impl/URLResourceReaderTest.java

```diff
@@ -708,31 +713,10 @@ public void testSetRootResourceDirectoriesInvalidPath() throws Exception {
     assertThat(rootResourceDirectories.size(), is(1));
   }
 
-  @Test
-  public void testSetRedirect() throws Exception {
```

Comment:
Why did we remove this test?

Reply:
The test doesn't make any sense with the way that I'm now configuring the client. This is tested elsewhere.